package.json

```diff
@@ -39,7 +39,7 @@
     "prepare-ghpages": "node ./scripts/docs/prepare.js",
     "remove-ghpages": "node ./scripts/docs/remove.js",
     "lint": "jshint scripts/ packages/ system-test/ test/ && jscs packages/ system-test/ test/",
-    "test": "npm run docs && npm run snippet-test && npm run unit-test",
+    "test": "rm -rf docs/json && npm run docs && npm run snippet-test && npm run unit-test",
```

Comment:
We might also need to do this programmatically in case `npm test` is run from the `packages` directory, the local path, "docs/json", might not locate the correct directory.